tests/test_session.py

```diff
@@ -42,3 +42,24 @@ def test_is_temporal_session_on_raw_session(self, session, connection):
                 assert not is_temporal_session(raw_session)
             finally:
                 raw_session.close()
+
+    def test_different_sessions_update_vclock(self, session, connection, sessionmaker, newstylemodel):
+        session.add(newstylemodel)
+        assert newstylemodel.vclock == 1
+        session.commit()
+
+        # create different session
+        transaction = connection.begin()
+        second_session = sessionmaker(bind=connection)
+        refreshed_model = second_session.query(models.NewStyleModel).first()
+
+        # update row within new session
+        refreshed_model.activity = models.Activity(description="Activity Description")
+        refreshed_model.description = "a new str"
+        second_session.add(refreshed_model)
+        assert refreshed_model.vclock == 2
+        second_session.commit()
+
+        # clear out db
+        transaction.rollback()
+        second_session.close()
```

Comment:
this is pretty neat, might also want to refetch the model from the first session and see that the vclock is two there